packages/clerk-js/src/core/resources/Session.ts

```diff
@@ -85,7 +85,8 @@ export class Session extends BaseResource implements SessionResource {
 
   getToken: GetToken = async (options?: GetTokenOptions): Promise<string | null> => {
     return runWithExponentialBackOff(() => this._getToken(options), {
-      shouldRetry: (error: unknown, currentIteration: number) => !is4xxError(error) && currentIteration < 4,
+      shouldRetry: (error: unknown, currentIteration: number) =>
```

Comment:
Do we really not want to retry on 5xx at all? Traditionally retrying on 5xx is common practice in the case of intermittent errors, but I do see the risk with potentially making the underlying problem worse.